### PR TITLE
Add seed corpus for woff2, add missing options for other fuzzer.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

**Comment by inferno-chromium** [comment]
@rsheeter - fyi, should fix #183

File: projects/woff2/Dockerfile [hunk 1]
```diff
@@ -20,4 +20,5 @@ RUN apt-get install -y make autoconf automake libtool
 
 RUN git clone --depth 1 --recursive https://github.com/google/woff2
 WORKDIR woff2
-COPY build.sh convert_woff2ttf_fuzzer.* $SRC/
+COPY build.sh *.options seed_corpus.zip $SRC/
```
Comment (Dor1s) [inline projects/woff2/Dockerfile]: Where this file comes from?

Comment (Dor1s) [inline projects/woff2/Dockerfile]: Oops, github doesn't show what I highlighted. I meant `seed_corpus.zip`. I don't see where we download it or anything else.

Comment (Dor1s) [inline projects/woff2/Dockerfile]: Ok, now I see! Wouldn't it be better to store testcases unpacked, as we do in Chromium?

Comment (inferno-chromium) [inline projects/woff2/Dockerfile]: Done!

**Comment by inferno-chromium** [comment]
Merging for now. Ideally @rsheeter, we should have all these test files inside woff2 repo, please move once you get a chance and add other seed corpus files that you have.

**Pull request merged**